NOTICE

```diff
@@ -9,7 +9,7 @@ The Apache Software Foundation (http://www.apache.org/).
 
 ================= Apache Calcite 1.10.0 =================
 Apache Calcite
-Copyright 2012-2016 The Apache Software Foundation
+Copyright 2012-2022 The Apache Software Foundation
```

Comment:
with the exception of Druid, i believe these years are for the version of the software included, though not sure it matters much